templates/countOfficersTemplate.php

```diff
@@ -0,0 +1,11 @@
+<?php include 'inc/header.php'; ?>
+<h2 class="page-header">Click Button to Show Count of All Officers</h2>
+<form method="GET" action="countofficers.php">
+    <input type="submit" name="show" class="btn btn-primary" value="Show Count Of Officers">
+</form>
+<div class=open-spaces>
+    <p><?php if ($show == 'Show Count Of Officers') {
```

Comment:
@Junkim97 try to to play around with the html/css make it standout a bit more instead of just a <p>